internal/cli/atlas/clusters/clusters_test.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build unit
+
+package clusters
+
+import (
+	"testing"
+
+	"github.com/mongodb/mongocli/internal/test"
+)
+
+func TestBuilder(t *testing.T) {
+	test.CmdValidator(
```

Comment:
[q] What is being tested here?

Reply:
we are testing that `Builder ()` in `clusters.go` has 11 subcommand and no flags